<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListView" name="lvSourse">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>191</width>
     <height>315</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>330</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Backup!</string>
   </property>
  </widget>
  <widget class="QListView" name="lvBackup">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>9</y>
     <width>191</width>
     <height>315</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
